robottelo/config/__init__.py

```diff
@@ -99,6 +99,18 @@ def get_url():
     return urlunsplit((scheme, hostname, '', '', ''))
 
 
+def get_ssl_cert_verify():
+    """Return the SSL certificate or setting to verify HTTPS requests.
+
+    :return: Certificate path or boolean verify setting
+    :rtype: bool or str
+    """
+    try:
+        return settings.server.verify_ca
```

Comment:
You can validate settings from https://github.com/SatelliteQE/robottelo/blob/master/robottelo/config/validators.py to return the `False` value by default and hence need not to handle exception here explicitly!

Reply:
Also, you don't need this function in that case, just directly call the settings object wherever applicable

Reply:
Thanks, in that case I will adapt it